src/helpers/themes/theming.css

```diff
@@ -36,22 +36,24 @@
     calc(var(--oe-theme-lightness) - 90%)
   );
 
-  --oe-border-rounding: 6px;
+  --oe-border-rounding: 1em;
   --oe-border-color: hsl(var(--oe-theme-hue), var(--oe-theme-saturation), calc(var(--oe-theme-lightness) - 10%));
-  --oe-border-width: 2px;
+  --oe-border-width: 1px;
 
-  --oe-box-shadow: 1px 1px 1px currentcolor;
+ 
   --oe-backdrop-shadow: 3px 3px 4px;
   --oe-font-family: "Roboto", sans-serif;
   --oe-font-size: 11px;
 
   --oe-animation-time: 0.3s;
-  --oe-spacing: 0.6em;
+  --oe-spacing: 0.8em;
+  --oe-spacing-l: 1em;
 
   --oe-primary-color: hsl(var(--oe-theme-hue), var(--oe-theme-saturation), var(--oe-theme-lightness));
   --oe-secondary-color: hsl(var(--oe-theme-hue), var(--oe-theme-saturation), calc(var(--oe-theme-lightness) + 25%));
   --oe-accent-color: hsl(var(--oe-theme-hue), var(--oe-theme-saturation), calc(var(--oe-theme-lightness) + 20%));
   --oe-info-color: hsl(207deg, calc(var(--oe-theme-saturation) + 45%), calc(var(--oe-theme-lightness) + 15%));
+  --oe-info-bg-color: hsl(207deg, calc(var(--oe-theme-saturation) + 45%), calc(var(--oe-theme-lightness) + 40%));
```

Comment:
I'd probably compute the background color from the info color so that if you change the `--oe-info-color` e.g. in our workbench client themeing, the background color is automatically updated.

```css
 --oe-info-bg-color: color-mix(in srgb, var(--oe-info-color) 15%, white);
```

Reply:
Thanks, will add this change